automates/text_reading/src/test/scala/org/clulab/aske/automates/text/TestFunctions.scala

```diff
@@ -243,7 +294,7 @@ class TestFunctions extends ExtractionTest {
   passingTest should s"find functions from t2g: ${t2g}" taggedAs(Somebody) in {
     val desired = Seq(
       "Es" -> Seq("supply of energy to the surface"), // fixme: one additional, unwanted concept is captured as an input due to bad parsing.
-      "water movement to the evaporating sites near the surface" -> Seq("hydraulic properties of the soil")
+      "water movement to the evaporating sites near the surface" -> Seq("hydraulic properties of the soil") // note: this test was broken after writing filterInputOverlaps action
```

Comment:
Is this by any chance resolved now?

Reply:
Oops, yes. It was already solved. Thanks for catching that!